hexa/databases/utils.py

```diff
@@ -1,28 +1,34 @@
 import psycopg2
 from psycopg2 import sql
 
-from hexa.plugins.connector_postgresql.models import Database
 from hexa.workspaces.models import Workspace
 
+from .api import get_db_server_credentials
 
-def get_workspace_database(workspace: Workspace):
-    return Database.objects.get(database="hexa-explore-demo")
 
+def get_database_url(database: str):
+    credentials = get_db_server_credentials()
+    role = credentials["role"]
+    password = credentials["password"]
+    host = credentials["host"]
+    port = credentials["port"]
+
+    return f"postgresql://{role}:{password}@{host}:{port}/{database}"
 
-def get_database_definition(workspace: Workspace):
-    database = get_workspace_database(workspace)
 
-    with psycopg2.connect(database.url) as conn:
+def get_database_definition(workspace: Workspace):
+    url = get_database_url(workspace.db_name)
+    with psycopg2.connect(url) as conn:
```

Comment:
Isn't it a problem that we do not close the connection here ? (if we do it above with the creation ?) cc: @pvanliefland

Reply:
Hum yes. Will need to add try ... finally block